.gitignore

```diff
@@ -6,6 +6,9 @@ __pycache__
 *.pyc
 *.log
 *.egg-info
+*,cover
```

Comment:
Maybe this comma should be a dot instead.

Reply:
When I executed `pytest --cov-report annotate`, somehow `*,cover` files were generated. But it is not generated when I run it now. So I'll delete this.

Reply:
c508d0e